pyresample/geometry.py

```diff
@@ -1904,8 +1904,10 @@ def __getitem__(self, key):
 def get_geostationary_angle_extent(geos_area):
     """Get the max earth (vs space) viewing angles in x and y."""
     # get some projection parameters
-    req = geos_area.proj_dict['a'] / 1000.0
-    rp = geos_area.proj_dict['b'] / 1000.0
+    from pyresample.utils import proj4_radius_parameters
```

Comment:
Is there an issue moving this up to the beginning of the file ?

Reply:
No, just that I wanted to avoid a needless import for the whole module. I can move it though.